README.md

```diff
@@ -808,5 +809,24 @@ Implemeted for a git session - 21/09/2019 (Better idea would have been to use in
 |||||
 |||||
 |||||
-|||||
-|807|Zeraora|Electric|![Zeraora](https://assets.pokemon.com/assets/cms2/img/pokedex/full/807.png)|
+|807|Zeraora|![Electric](https://vignette.wikia.nocookie.net/pokemon/images/a/aa/Type_Electric.gif)|![Zeraora](https://assets.pokemon.com/assets/cms2/img/pokedex/full/807.png)|
+
+Link Legends for the Pokemon types :-
+https://vignette.wikia.nocookie.net/pokemon/images/6/64/Type_Bug.gif
+https://vignette.wikia.nocookie.net/pokemon/images/0/0d/Type_Dark.gif
+https://vignette.wikia.nocookie.net/pokemon/images/2/26/Type_Dragon.gif
+https://vignette.wikia.nocookie.net/pokemon/images/a/aa/Type_Electric.gif
+https://vignette.wikia.nocookie.net/pokemon/images/7/74/Type_Fairy.gif
+https://vignette.wikia.nocookie.net/pokemon/images/6/6b/Type_Fighting.gif
+https://vignette.wikia.nocookie.net/pokemon/images/4/4d/Type_Fire.gif
+https://vignette.wikia.nocookie.net/pokemon/images/4/4b/Type_Flying.gif
+https://vignette.wikia.nocookie.net/pokemon/images/f/f2/Type_Ghost.gif
+https://vignette.wikia.nocookie.net/pokemon/images/4/46/Type_Grass.gif
+https://vignette.wikia.nocookie.net/pokemon/images/1/1d/Type_Ground.gif
+https://vignette.wikia.nocookie.net/pokemon/images/8/84/Type_Ice.gif
+https://vignette.wikia.nocookie.net/pokemon/images/6/61/Type_Normal.gif
+https://vignette.wikia.nocookie.net/pokemon/images/8/82/Type_Poison.gif
+https://vignette.wikia.nocookie.net/pokemon/images/6/65/Type_Psychic.gif
+https://vignette.wikia.nocookie.net/pokemon/images/b/b3/Type_Rock.gif
+https://vignette.wikia.nocookie.net/pokemon/images/a/ab/Type_Steel.gif
+https://vignette.wikia.nocookie.net/pokemon/images/e/ed/Type_Water.gif
```

Comment:
Can you turn it to a list with the icons only as they look not the links, people can just copy the whole thing

Reply:
done